src/intTest/java/uk/nhs/digital/nhsconnect/lab/results/inbound/InboundMeshQueueMultiMessageTest.java

```diff
@@ -58,19 +63,15 @@ void whenMeshInboundQueueMessageIsReceivedThenMessageIsHandled(SoftAssertions so
 
     @SuppressWarnings("checkstyle:magicnumber")
     private void assertGpOutboundQueueMessages(SoftAssertions softly) throws IOException, JMSException, JSONException {
-        final List<Message> gpOutboundQueueMessages = IntStream.range(0, 6)
+        final List<Message> gpOutboundQueueMessages = IntStream.range(0, 2)
             .mapToObj(x -> getGpOutboundQueueMessage())
             .collect(Collectors.toList());
```

Comment:
Not sure the streaming is valuable anymore
```suggestion
        final List<Message> gpOutboundQueueMessages = List.of(getGpOutboundQueueMessage(), getGpOutboundQueueMessage());
```

Reply:
Good spot - thanks!